setup.py

```diff
@@ -38,6 +34,8 @@
         test=[
             'pytest-cov',
             'WebTest >= 2.0.14'
+            'tox',
+            'flake8'
         ],
```

Comment:
As mentioned in the other repos - flake8 should not be necessary, as it gets installed via tox (deps=...).